lessons/Early-Math/translating/langs/en-us/index.adoc

```diff
@@ -0,0 +1,183 @@
+- Take a look at this sentence: "My heroes are my grandparents, Batman and Wonder Woman."
+- How do you interpret the sentence?
+** _If students do not giggle at the suggestion that your grandparents are superheroes, try re-reading the sentence with different inflections - or invite a student to do so. Discuss the two different possible interpretations of the sentence._
+
+Math is precise, but that precision is difficult to preserve when we switch to words. Often, sentences can be _ambiguous_, meaning that there is more than one way to interpret them!
+
+One reason that Circles of Evaluation are so powerful is that they eliminate the ambiguity we encounter when representing expressions with words.  They also delineate expressions more clearly than traditional mathematical notation. In this lesson, we tackle expressions in words that have _more than one_ possible mathematical translation.
+
+=== Investigate
+
+Take a look at this expression: "the sum of three and two multiplied by eight"
+
+Are we multiplying first and then adding (as represented by the Circle on the left), or adding first and then multiplying (as represented by the Circle on the right)?
+
+[.embedded, cols="^.^1,^.^1", grid="none", stripes="none" frame="none"]
+|===
+
+|@show{(coe  '(+ 3 (* 2 8)))}		| @show{(coe  '(* (+ 3 2) 8))}
+|===
+
+In this case, there are multiple ways to translate the sentence. (Would inserting a comma after the word “two” provides clarity?)
+
+[.lesson-instruction]
+- Complete @printable-exercise{ambiguity-of-words.adoc}, drawing two possible Circles for each expression in words.
+- What happens when you translate each Circle into a mathematical expression? Do the expressions produce the same result?
+** _The expressions are structurally different, and generally produce different results (with two noteworthy exceptions!)_
+- Did you notice anything interesting about the last two expressions, compared to the others on the page?
+** _These expressions use only multiplication or only addition. As a result, they are equivalent - meaning that they evaluate to the same result regardless of which Circle we use._
+
+We don't want students to think _all_ mathematical expressions in words are ambiguous, as that is simply not the case! Emphasize that only certain verbal structures create this confusion; some phrases are indeed clearer than others. The following activity emphasizes this idea.
+
+[.lesson-instruction]
+- On @printable-exercise{ambiguous-or-clear.adoc}, identify the expressions that have two different numeric translations.
+- When you encounter an expression that is ambiguous, rewrite it two times - once for each possible interpretation.
+- When you encounter an expression that is clear, draw its Circle of Evaluation.
+
+=== Synthesize
+
+- Why are some expressions in words ambiguous and others are not?
+- Do you think that expressions written in the language of math have ambiguity?
+- Are Circles of Evaluation ever ambiguous?
+
```

Comment:
Noting that, while @schanzer is fairly attached to this glossary definition,

> **circle of evaluation** a 'sentence diagram' of the structure of a mathematical expression

I, personally, think it's doing us a disservice because I love circles of evaluation and do not find sentence diagrams to be helpful or inspiring... nor do I think most students in the 21st century have any exposure to them.

Don't necessarily have energy to think this one through, and it's not a hill I have energy to defend, but encourage y'all to discuss. Any reason it can't just be described as a diagram (which most people recognize as a helpful tool)... rather than a sentence diagram?